packages/cli/src/parse-command-line.ts

```diff
@@ -24,6 +25,9 @@ export const parseCommandLine = () => {
 	if (parsedCli.png) {
 		Config.Output.setOutputFormat('png-sequence');
```

Comment:
Let's deprecate the PNG flag and undocument it, because it's redundant now.

If user passes CLI flag `--png`, let's log `--png has been deprecated, use --format=png from now on` (but let's keep the user continue for now)

Reply:
yeah that is needed I guess. Let's keep --png flag for 2 more version. then we can just drop the support for png flag.

Reply:
sounds good. or we can release like version 2.0 with audio support, and drop all the deprecated stuff 👍

Reply:
yeah I was thinking the same.